src/bp/migrations/v12_24_0-1625965172-remove_channels_from_botpress_config.ts

```diff
@@ -0,0 +1,53 @@
+import * as sdk from 'botpress/sdk'
+import { ModuleConfigEntry } from 'core/config'
+import { Migration, MigrationOpts } from 'core/migration'
+import _ from 'lodash'
+
+const EXCEPTION = 'channel-web'
+
+const migration: Migration = {
+  info: {
+    description: 'Remove channels from list of modules in botpress config',
+    target: 'core',
+    type: 'config'
+  },
+  up: async ({ configProvider }: MigrationOpts): Promise<sdk.MigrationResult> => {
+    const config = await configProvider.getBotpressConfig()
+    const modules: ModuleConfigEntry[] = []
+
+    for (const module of config.modules) {
+      const { location, enabled } = module
+
+      if (!location.includes('channel-') || location.includes(EXCEPTION)) {
+        modules.push({ location, enabled })
+      }
```

Comment:
Please provide a full list of modules to remove instead of that exception. If someone created a channel it would be removed by that logic